src/Sylius/Component/Core/OrderProcessing/OrderShipmentProcessor.php

```diff
@@ -104,4 +104,20 @@ private function getOrderShipment(BaseOrderInterface $order)
             return null;
         }
     }
+
+    /**
+     * @param OrderInterface $order
+     *
+     * @return bool
+     */
+    private function ifItemsRequireShipping(OrderInterface $order)
```

Comment:
This method is duplicated [here](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Core/Checker/OrderShippingMethodSelectionRequirementChecker.php#L71), perhaps we extract it to a service to respect SRP?

Can be done as a separate PR.